projects/ngrx.io/content/marketing/resources.json

```diff
@@ -148,6 +148,12 @@
               "url": "https://co-it.gitbook.io/ngrx-ducks/",
               "rev": true
           },
+          "ngrx-handlers": {
+            "title": "ngrx-handlers",
+            "desc": "A plugin for boilerplate reduction",
```

Comment:
Can we think of a description without calling it boilerplate?

For example, I like how the redux toolkit describe the `createSlice` method, which is more or less what your library provides (if I'm not mistaken):

```
A function that accepts an initial state, an object full of reducer functions, and a "slice name", and automatically generates action creators and action types that correspond to the reducers and state.
```

Reply:
Sure 🙂

Does `A plugin for generating actions and reducer based on the handler map` sounds better?

Reply:
@timdeschryver 

I updated description with suggested text. Also, I updated the docs: https://github.com/markostanimirovic/ngrx-handlers

Reply:
Thanks @marcinmajkowski !